mRemoteV1/App/Startup.cs

```diff
@@ -282,29 +309,13 @@ private static void ParseCommandLineArgs()
                     NoReconnectParam = "norc";
                 }
 
-                if (!string.IsNullOrEmpty(ConsParam))
+                // Handle custom connection file location
+                Settings.Default.CustomConsPath = GetCustomConsPath(ConsParam);
+                if (Settings.Default.CustomConsPath != null)
                 {
-                    if (File.Exists(cmd[ConsParam]) == false)
-                    {
-                        if (File.Exists(GeneralAppInfo.HomePath + "\\" + cmd[ConsParam]))
-                        {
-                            Settings.Default.LoadConsFromCustomLocation = true;
-                            Settings.Default.CustomConsPath = GeneralAppInfo.HomePath + "\\" + cmd[ConsParam];
-                            return;
-                        }
-                        if (File.Exists(ConnectionsFileInfo.DefaultConnectionsPath + "\\" + cmd[ConsParam]))
-                        {
-                            Settings.Default.LoadConsFromCustomLocation = true;
-                            Settings.Default.CustomConsPath = ConnectionsFileInfo.DefaultConnectionsPath + "\\" + cmd[ConsParam];
-                            return;
-                        }
-                    }
-                    else
-                    {
-                        Settings.Default.LoadConsFromCustomLocation = true;
-                        Settings.Default.CustomConsPath = cmd[ConsParam];
-                        return;
-                    }
+                    Settings.Default.LoadConsFromCustomLocation = true;
+                    Settings.Default.Save();
+                    return;
```

Comment:
@kmscode I'm returning here, before saving the settings file, simply because this was done as such in [`Startup.cs`](https://github.com/mRemoteNG/mRemoteNG/blob/d15e444cb79aeb8f6e0b90b5687276e71ddc1400/mRemoteV1/App/Startup.cs) lines [293](https://github.com/mRemoteNG/mRemoteNG/blob/d15e444cb79aeb8f6e0b90b5687276e71ddc1400/mRemoteV1/App/Startup.cs#L293), [299](https://github.com/mRemoteNG/mRemoteNG/blob/d15e444cb79aeb8f6e0b90b5687276e71ddc1400/mRemoteV1/App/Startup.cs#L299), and [306](https://github.com/mRemoteNG/mRemoteNG/blob/d15e444cb79aeb8f6e0b90b5687276e71ddc1400/mRemoteV1/App/Startup.cs#L306) as of d15e444cb79aeb8f6e0b90b5687276e71ddc1400, which is where I'm branching from.

But I don't think that is correct, there is a bunch of stuff that can be set by the console parameters, and since we return here, we're basically ignoring them. Is this intended?

Reply:
I agree. That return should be removed. 

Fix that and I'll accept the PR.

Thanks!

Reply:
Will do, expect that in a bit.